pywbem/cim_operations.py

```diff
@@ -1255,14 +1331,404 @@ class are to be included in the returned instances, as follows:
+
+          PropertyList (:term:`py:iterable` of :term:`string`):
+            An iterable specifying the names of the properties to be
+            included in the returned instances, in any lexical case.
+
+            An empty iterable indicates to include no properties.
+
+            If `None`, all properties are included.
+
+          FilterQueryLanguage (:term:`string`):
+            Optional: a String defining the name of the query language
+            used for the Query argument. This argument must exist if the
+            filterQuery argument is provided.
+
+          QueryLanguage (:term:`string`):
+            Optional: a String defining the query that is to be sent
+            to the WBEM server.
+          #TODO define typing for this correctly
+          OperationTimeout (term: `Uint32`):
+            Optional: If provided it defines a timeout value to be
+            sent to the WBEM Server
+            * If `Uint32` value, this value is sent to the server as the
+              proposed timeout for the operation sequence. If the client
+              waits longer than this timeout in seconds, the server may
+              terminate the operation
+            * if `None`
+
+          ContinueOnError (:class:`py:bool`):
+            Indicates to the server to continue sending responses
+            after errorresponse has been sent.
+            * if `True` the server may continue sending responses after
+              sending an error response.
+            * If `False` the server must not continue sending responses after
+              and error response is sent and must close the enumeration
+              sequence after sending any error response.
+            * if `None` this parameter is not sent to the WBEM server and the
+              WBEM server is to close the enumeration sequence upon error.
+
+          MaxObjectCount (term: `Uint32`)
+            Defines a value that sets the maximum number of instances the
+            WBEM server can return for this request. any Uint32 number including
+            zero is valid.
+            * If non-zero `Uint32` the WBEM server must return no more than the
+              number of instances defined by this value.
+            * If zero the WBEM server must return zero instances.  This may
+              be used by a client to restart the timeout.
+            * If `None`  a value of zero is sent to the WBEM server.
+
+          :Returns:
+
+            A tuple containing:
+            * A list of :class:`~pywbem.CIMInstance` objects
+              that are representations of the enumerated instances.
+            * instance of enumeration_context which contains
+              information about the state returned by the successfully
+              completed operation. This object may be queried with
+              the more() method to determine if this response
+              terminates the enumeration sequence.
+
```

Comment:
I don' t like the merging of the output params into EnumContext. Can we not have the same output params asin DSYP0200, returned as a tuple? For consistency with the spec. that's how we do it for the other intrjnsic ops as well.